lib/list_widget.dart

```diff
@@ -0,0 +1,57 @@
+import 'package:flutter/material.dart';
+
+///ListWidget: 
+///  - a widget representing a page containing a single list in the 
+///  app, such as a to-do list, a shopping list, etc.
+
+class ListWidget extends StatefulWidget {
```

Comment:
```suggestion
/// ListWidget: 
///  - a widget representing a page containing a single list in the 
///  app, such as a to-do list, a shopping list, etc.
class ListWidget extends StatefulWidget {
```
Don't leave blank line after the comment, be sure to leave a space between the `///` and your words.